fatiando/gravmag/transform.py

```diff
@@ -262,6 +263,61 @@ def tga(x, y, data, shape, method='fd'):
     return res
 
 
+def tilt(x, y, data, shape, xderiv=None, yderiv=None, zderiv=None):
+    """
+    Calculates the potential field tilt, as defined by Miller and Singh (1994):
+
+    .. math::
+        tilt(f) = tan^{-1}(\\frac{\\frac{dT}{dz}}{\\sqrt{\\frac{dT}{dx}^2 +
+        \\frac{dT}{dy}^2}})
+
+    .. note:: Requires gridded data.
+
+    Parameters:
+
+    * x, y : 1D-arrays
+        The x and y coordinates of the grid points
+    * data : 1D-array
+        The potential field at the grid points
+    * shape : tuple = (nx, ny)
+        The shape of the grid
+    * xderiv : 1D-array
+        Optional. Values of the derivative in the x direction
+    * yderiv : 1D-array
+        Optional. Values of the derivative in the y direction
+    * zderiv : 1D-array
+        Optional. Values of the derivative in the z direction
+
+    Returns:
+
+    * tilt : 1D-array
+        The tilt angle of the total field.
+
+    References:
+
+    Miller, Hugh G, and Vijay Singh. 1994. "Potential Field Tilt --- a New
+    Concept for Location of Potential Field Sources."
+    Journal of Applied Geophysics 32 (2--3): 213-17.
+    doi:10.1016/0926-9851(94)90022-1.
+    """
+    #Deal with horizontal derivatives. If we have not been given an array
+    #with either, we will calculate them first, then get the horizontal
+    #derivative.
+    if xderiv == None:
```

Comment:
Avoid comparing to `None` with `==` or `!=`. The proper way is to use `xderiv is None` or `xderiv is not None`.